cloudslang_dsl_reference.rst

```diff
@@ -853,7 +852,9 @@ input name may in turn be mapped to its properties or an input
 `expression <#expressions>`__.
 
 Inputs are used to pass parameters to `flows <#flow>`__ or
-`operations <#operation>`__.
+`operations <#operation>`__. Input names for a specific `flow <#flow>`__ or
+`operation <#operation>`__ must be different than the `output <#outputs>`__
+names of the same `flow <#flow>`__ or `operation <#operation>`__.
```

Comment:
Are the step argument names not the same as the calling flow/operation inputs? So if a flow/operation's inputs and output  names are different, then the step argument names will be different as well.